client/my-sites/email/form/mailboxes/constants.ts

```diff
@@ -10,6 +10,7 @@ export const FIELD_IS_ADMIN = 'isAdmin';
 export const FIELD_MAILBOX = 'mailbox';
 export const FIELD_NAME = 'name';
 export const FIELD_PASSWORD = 'password';
+export const FIELD_PASSWORD_RESET_EMAIL = 'passwordResetEmail';
```

Comment:
Sorry if I was a bit unclear in my previous review, but what I meant was that me could **replace** `FIELD_ALTERNATIVE_EMAIL` with `FIELD_PASSWORD_RESET_EMAIL`. That way, we have a single constant to represent this type of field (without the naming confusion that "alternate"/"alternative" email brings).

The only place we really need to differentiate is when we serialize that field to be sent to the back-end. It wouldn't be very complex to use a different key (`alternative_email` or `recoveryEmail`) depending on whether the provider is Titan or Google.

Reply:
I like this approach very much 👍. Ideally, we should also update other parts of our code to follow the same naming convention (e.g. rename `AlternateEmailValidator` to `PasswordResetEmailValidator`).

Reply:
@AllTerrainDeveloper I see that we've already gone through the exercise to replace all occurrences of `FIELD_ALTERNATIVE_EMAIL` to `FIELD_PASSWORD_RESET_EMAIL` in this PR, should we also rename `AlternateEmailValidator` to `PasswordResetEmailValidator` as @stephanethomas flagged?

Reply:
I agree, we should address @wongasy's comment to avoid naming confusion